verl/workers/megatron_workers.py

```diff
@@ -566,7 +681,18 @@ async def rollout_mode(self):
             log_gpu_memory_usage("After load actor params during rollout_mode", logger=logger)
 
         if self.bridge is not None:
-            per_tensor_param = self.bridge.export_weights(self.actor.actor_module)
+            if self.provider is None:
+                per_tensor_param = self.bridge.export_weights(self.actor.actor_module)
+            else:
+                if self.peft_config is not None:
+                    logger.warning(
+                        "Exporting HF weights with PEFT adapters hasn't been supported in Megatron-Bridge yet. "
+                        "Now we just skip exporting weights for debugging purpose."
+                    )
+                    # TODO: Fix this
+                    per_tensor_param = (x for x in [])
```

Comment:
This is something not implemented in Megetron-Bridge, we need to support this so that vLLM's adapters' weight can be properly updated. I have opened an issue https://github.com/NVIDIA-NeMo/Megatron-Bridge/issues/1272 which is directly related to this.

Reply:
PR opened at the Megatron-Bridge side, this should be resolved if we use that PR: https://github.com/NVIDIA-NeMo/Megatron-Bridge/pull/1310 Feel free to let me know if you have any comments.